src/components/nowPlayingBar/nowPlayingBar.js

```diff
@@ -501,21 +534,30 @@ define(['require', 'datetime', 'itemHelper', 'events', 'browser', 'imageLoader',
         var nowPlayingItem = state.NowPlayingItem;
 
         var textLines = nowPlayingItem ? nowPlayingHelper.getNowPlayingNames(nowPlayingItem) : [];
-        if (textLines.length > 1) {
-            textLines[1].secondary = true;
-        }
-        nowPlayingTextElement.innerHTML = textLines.map(function (nowPlayingName) {
-
-            var cssClass = nowPlayingName.secondary ? ' class="nowPlayingBarSecondaryText"' : '';
-
-            if (nowPlayingName.item) {
-                var nowPlayingText = getTextActionButton(nowPlayingName.item, nowPlayingName.text);
-                return `<div ${cssClass}>${nowPlayingText}</div>`;
+        nowPlayingTextElement.innerHTML = '';
+        if (textLines) {
+            let secondaryText = document.createElement('div');
+            secondaryText.classList.add('nowPlayingBarSecondaryText');
+            let itemText = document.createElement('div');
+            if (textLines.length > 1) {
+                textLines[1].secondary = true;
+                if (textLines[1].text) {
+                    let text = document.createElement('a');
+                    text.innerHTML = textLines[1].text;
+                    secondaryText.appendChild(text);
+                }
             }
 
-            return `<div ${cssClass}>${nowPlayingText}</div>`;
-
-        }).join('');
+            if (textLines[0].text) {
+                if (textLines[0].text) {
```

Comment:
overdid :p
also you could leave creation order: `itemText`, `secondaryText`

Reply:
@dmitrylyzo Lol, not sure what I was thinking here :P